legion_airflow/legion_airflow/hooks/s3_hook.py

```diff
@@ -194,6 +212,98 @@ def copy_folder(self, src_bucket: str, src_key: str, dest_bucket: str, dest_key:
                                  .format(self.bucket_prefix + src_bucket, key_from, dest_bucket, key_to))
                 dist_obj.copy(source)
 
+    def load_file(self,
```

Comment:
load here is a bit confusing. Would it be better to call it store() or upload()?